charts/4allportal/templates/webdav/configmap.yaml

```diff
@@ -124,4 +127,63 @@ data:
     BrowserMatch " Konqueror/4" redirect-carefully
     BrowserMatch "^gvfs" redirect-carefully
     BrowserMatch "WebDAVLib" redirect-carefully
+  log-handler.sh: |
+    #!/bin/ash
+    ENDPOINT="http://{{ include "common.names.fullname" . }}-backend/api/extensions/dam/fs-event"
+    while IFS= read -r line; do
+      status=$(echo "$line" | cut -d';' -f1)
+      type=$(echo "$line" | cut -d';' -f2)
+      path=$(echo "$line" | cut -d';' -f3)
```

Comment:
$path and $dest variables need to be relativized to $mount variable.
What I mean is that the path has to be relative to the 'prefix' which is pre-defined in $mount variable

Reply:
In WebDAV we can see two default points: `/webdav/assets/` and `/webdav/data/`
Should we ignore `data` in that case?

Reply:
Yes, data can be ignored for the purpose of webdav events.

Reply:
Done